pkg/apis/dashboard/migration/conversion/conversion_test.go

```diff
@@ -0,0 +1,41 @@
+package conversion
+
+import (
+	"fmt"
+	"testing"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/stretchr/testify/require"
+
+	dashboard "github.com/grafana/grafana/pkg/apis/dashboard"
+	dashboardV0 "github.com/grafana/grafana/pkg/apis/dashboard/v0alpha1"
+	dashboardV1 "github.com/grafana/grafana/pkg/apis/dashboard/v1alpha1"
+	dashboardV2 "github.com/grafana/grafana/pkg/apis/dashboard/v2alpha1"
+)
+
+func TestConversionMatrixExist(t *testing.T) {
```

Comment:
The downside to explicit conversions is that we may not define all combinations... this test helps ensure that all flavors exist